features/wishlists.feature

```diff
@@ -27,3 +27,11 @@ Scenario: Create a Wishlist
     And I should see "Beckett's Halloween Costume List" in the "Name" field
     And I should see "42" in the "Customer ID" field
     And "Is Public" should "be" checked
+
+Scenario: Delete a Wishlist
+    When I visit the "Admin UI Page"
```

Comment:
Hey @raymondlu8547, good start but a couple things to update.  #1 I think we will need to create a wishlist first before we test the delete.  We designed our wishlist API to return a 204 No Content response when deleting a non-existent wishlist_id.  (Which is what happens in this case).  In the $.ajax().done() method we should check the response code and perhaps give a different message for each.  Like "Wishlist with ID [5432] successfully deleted" for a 200 response and maybe "Could not find a wishlist to delete with ID [42]" when the response is 204.

So in your behave test you should:
1. Create a wishlist
2. Read the new ID 
3. Try to delete the newly created ID
4. Assert that the message is the 200 success message
5. Try to delete the same ID a second time
6. Assert that the message is the 204 message